### PR TITLE
Closes #106, #107 - New comment form

### LINKED ISSUE #106: Lister Leaves Comment For Contractor
As a logged in lister,
Who has a job that is "Completed",
When I visit "/users/<my_username>/jobs/<job_id>/comments/new",
And I pick a rating (1 - 5),
And I enter a comment greater than 50 characters,
And I click "Leave Comment",
Then I am taken to my dashboard,
And if I visit "/users/<contractor_username>,
Then I see my comment for the contractor under "Previous Jobs" which includes:
- comment text
- my name as a link to my public profile
- comment rating
- comment posted at date
- accepted bid price

### LINKED ISSUE #107: Contractor Leaves Comment For Lister
As a logged in contractor,
Who has worked on a job that is now "Completed",
When I visit "/users/<lister_username>/jobs/<job_id>/comments/new",
And I pick a rating (1 - 5),
And I enter a comment greater than 50 characters,
And I click "Leave Comment",
Then I am taken to my dashboard,
And if I visit "/users/<lister_username>,
Then I see my comment for the lister under "Previous Listings" which includes:
- comment text
- my name as a link to my public profile
- comment rating
- comment posted at date
- accepted bid price

### PR DESCRIPTION
**Pull request by ToniRib**
This branch sets up the form to leave feedback for another user (aka create a comment). It doesn't follow a workflow of staring at the dashboard since that is not in master yet, but goes directly to the new comment path. It includes 2 tests - one for the lister leaving a comment for the contractor, and one for the contractor leaving a comment for the lister.

closes #106 
closes #107

### CODE REVIEW

File: app/controllers/user/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class User::CommentsController < ApplicationController
```
Comment (ToniRib) [inline app/controllers/user/comments_controller.rb:1]: Ignoring this per our style decision

File: app/controllers/user/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @comment = Comment.new
+  end
+
+  def create
+    comment = current_user.comments.new(comment_params)
+
+    if comment.save
+      flash[:success] = "Comment Saved!"
+      redirect_to dashboard_path
+    else
```
Comment (ToniRib) [inline app/controllers/user/comments_controller.rb:16]: Ignoring this...it isn't redundant. There's stuff in there for sad path.

File: app/views/shared/_item_card.html.erb [hunk 3]
```diff
@@ -1,40 +0,0 @@
...
-
-        <% if item.active? %>
-          <h5 class="active-status-text text-accent-4">Status: <%= item.status %></h5>
-        <% else %>
-          <h5 class="inactive-status-text">Status: <%= item.status %></h5>
-        <% end %>
-      <% end %>
-    </div>
-  </div>
-</div>
```
Comment (jwperry) [inline app/views/shared/_item_card.html.erb:40]: Good catch, definitely need to scrap these files now.

**Comment by jwperry** [comment]
Looks great!

**Pull request merged**